test/testfloat/CMakeLists.txt

```diff
@@ -0,0 +1,157 @@
+# Fizzy: A fast WebAssembly interpreter
+# Copyright 2020 The Fizzy Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+option(FIZZY_TESTFLOAT_REQUIRED "The testfloat_gen tool is expected to be found" FALSE)
+
+if(FIZZY_TESTFLOAT_REQUIRED OR "$ENV{FIZZY_TESTFLOAT_REQUIRED}")
+    set(TESTFLOAT_REQUIRED TRUE)
+endif()
+
+add_executable(fizzy-testfloat testfloat.cpp)
+target_link_libraries(fizzy-testfloat PRIVATE fizzy::fizzy fizzy::test-utils)
+target_include_directories(fizzy-testfloat PRIVATE ${PROJECT_SOURCE_DIR}/lib)
+
+find_program(TESTFLOAT_GEN testfloat_gen)
+if(TESTFLOAT_GEN)
+    # TODO: Use random seed
+    # TODO: Make level explicit, add option to test level 2.
```

Comment:
What is level?

Reply:
`-level 1` - less tests; `-level 2` - more tests.

Reply:
So perhaps move this comment above the `set(TESTFLOAT_LEVEL 1` line.

Reply:
Ah, ok. This is already solved.